Gemfile.lock

```diff
@@ -60,9 +60,16 @@ GEM
       msgpack (~> 1.0)
     builder (3.2.4)
     byebug (11.1.1)
```

Comment:
we're not using byebug, so we should remove the gemfile.lock file and re-bundle.

Reply:
Done